src/components/scenes/DuressModeSettingScene.tsx

```diff
@@ -37,26 +34,6 @@ export const DuressModeSettingScene = (props: Props) => {
 
   const canDuressLogin = useWatch(account, 'canDuressLogin')
 
-  // Subscribe to pin event from VerifyPinScene
-  useEffect(
-    () =>
-      onPin(pin => {
```

Comment:
You forgot to delete the `export const [onPin, emitPin] = makeEvent<string>()` in the other file